service/src/main/kotlin/fi/espoo/evaka/shared/async/AsyncJobRunner.kt

```diff
@@ -141,6 +142,7 @@ class AsyncJobRunner<T : AsyncJobPayload>(val payloadType: KClass<T>, private va
                 remaining -= 1
                 config.throttleInterval?.toMillis()?.run { Thread.sleep(this) }
             } while (job != null && remaining > 0)
+            runningCount.decrementAndGet()
```

Comment:
This should be in a finally block, because if any exception is thrown (e.g. `it.claimJob` fails), the count gets messed up.